__tests__/app.test.js

```diff
@@ -211,4 +211,66 @@ describe('app.js', () => {
             })
         })
     })
+
+    describe('POST /api/articles/:article_id/comments', () => {
+        test('201: adds an item into the comments table with the correct keys', () => {
+            const input = {
+                username: 'lurker',
+                body: 'example body'
+            }
+            return request(app).post('/api/articles/1/comments').send(input)
+            .expect(201)
+            .then(({body}) => {
+                const {comment} = body
+                expect(typeof comment.comment_id).toBe('number')
+                expect(typeof comment.body).toBe('string')
+                expect(typeof comment.article_id).toBe('number')
+                expect(typeof comment.author).toBe('string')
+                expect(typeof comment.votes).toBe('number')
+                expect(typeof comment.created_at).toBe('string')
```

Comment:
you know exactly what article id, body and author should be based on the input of the send